### PR TITLE
[pt-br] Add docs/tasks/configure-pod-container/enforce-standards-admission-controller.md

### PR DESCRIPTION
**Pull request by DonatoHorn**
…standards-admission-controller.md

### CODE REVIEW

File: content/pt-br/docs/tasks/configure-pod-container/enforce-standards-admission-controller.md [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+    # - "latest" (default) 
+    # - specific version like "v{{< skew currentVersion >}}"
+    defaults:
+      enforce: "privileged"
+      enforce-version: "latest"
+      audit: "privileged"
+      audit-version: "latest"
+      warn: "privileged"
+      warn-version: "latest"
+    exemptions:
```
Comment (devlware) [inline content/pt-br/docs/tasks/configure-pod-container/enforce-standards-admission-controller.md:54]: Hi @DonatoHorn, its ok for you to leave the comments in the object in english. Could you revert the changes please?

**Review by devlware — CHANGES_REQUESTED** [review]

**Comment by DonatoHorn** [comment]
Hi, devlware, thanks for reviewing.

**Comment by devlware** [comment]
@stormqueen1990 could you stamp this one too? tks

File: content/pt-br/docs/tasks/configure-pod-container/enforce-standards-admission-controller.md [hunk 2]
```diff
@@ -0,0 +1,101 @@
+---
+title: Aplicando os Padrões de Segurança do Pod, Configurando o Controlador de Admissão Embutido
+content_type: task
+min-kubernetes-server-version: v1.22
+---
+
+Desde a versão v1.22, o Kubernetes fornece um [controlador de admissão]
+(/docs/reference/access-authn-authz/admission-controllers/#podsecurity)
+embutido para fazer cumprir os [padrões de segurança do Pod]
+(/docs/concepts/security/pod-security-standards).
```
Comment (stormqueen1990) [inline content/pt-br/docs/tasks/configure-pod-container/enforce-standards-admission-controller.md:10]: Both links are broken in the rendered page.

File: content/pt-br/docs/tasks/configure-pod-container/enforce-standards-admission-controller.md [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+Você pode configurar esse controlador de admissão para definir padrões em todo 
+o cluster e [excessões](/docs/concepts/security/pod-security-admission/#exemptions).
```
Comment (stormqueen1990) [inline content/pt-br/docs/tasks/configure-pod-container/enforce-standards-admission-controller.md:12]: Typo: did you mean "isenções" or "exceções" instead of "excessões"?

File: content/pt-br/docs/tasks/configure-pod-container/enforce-standards-admission-controller.md [hunk 4]
```diff
@@ -0,0 +1,101 @@
+---
+title: Aplicando os Padrões de Segurança do Pod, Configurando o Controlador de Admissão Embutido
```
Comment (stormqueen1990) [inline content/pt-br/docs/tasks/configure-pod-container/enforce-standards-admission-controller.md:2]: Thought: the original title is "Enforce Pod Security Standards by Configuring the Built-in Admission Controller". This connection between the two elements was lost in the translation.
Consider changing it to "Aplicando os Padrões de Segurança do Pod _Através_ da Configuração do Controlador de Admissão Embutido"

File: content/pt-br/docs/tasks/configure-pod-container/enforce-standards-admission-controller.md [hunk 5]
```diff
@@ -0,0 +1,101 @@
...
+Você pode configurar esse controlador de admissão para definir padrões em todo 
+o cluster e [excessões](/docs/concepts/security/pod-security-admission/#exemptions).
+
+## {{% heading "prerequisites" %}}
+
+{{% version-check %}}
+
+- Garanta que a `PodSecurity` do [portal de funcionalidade]
+(/docs/reference/command-line-tools-reference/feature-gates/#feature-gates-for-alpha-or-beta-features) 
+está ativada.
```
Comment (stormqueen1990) [inline content/pt-br/docs/tasks/configure-pod-container/enforce-standards-admission-controller.md:20]: This link is also broken. Markdown syntax does not allow for breaking lines between `[]` and `()`.

Another thought: I think we should refrain from translating feature gate. It would be better to keep it here but format in italic.

File: content/pt-br/docs/tasks/configure-pod-container/enforce-standards-admission-controller.md [hunk 6]
```diff
@@ -0,0 +1,101 @@
...
+      # Array of authenticated usernames to exempt.
+      usernames: []
+      # Array of runtime class names to exempt.
+      runtimeClasses: []
+      # Array of namespaces to exempt.
+      namespaces: []
+```
+
+{{< note >}}
+A v1beta1 de configuração requer v1.23+. Para v1.22, use v1alpha1.
```
Comment (stormqueen1990) [inline content/pt-br/docs/tasks/configure-pod-container/enforce-standards-admission-controller.md:61]: Suggestion: rephrase it in a way that makes it super clear what versions are those. Something like:
"A versão da configuração v1beta1 requer a versão v1.23 ou superior do Kubernetes. Para a versão v1.22 do Kubernetes, utilize v1alpha1."

**Review by stormqueen1990 — CHANGES_REQUESTED** [review]
Hi @DonatoHorn, thanks for your contribution! 🎉 

I found some broken links in the rendered version. Added some textual suggestions as well.

**Comment by DonatoHorn** [comment]
Thanks for reviewing.

File: content/pt-br/docs/tasks/configure-pod-container/enforce-standards-admission-controller.md [hunk 7]
```diff
@@ -0,0 +1,98 @@
...
+Desde a versão v1.22, o Kubernetes fornece um [controlador de admissão](/docs/reference/access-authn-authz/admission-controllers/#podsecurity)
+embutido para fazer cumprir os [padrões de segurança do Pod](/docs/concepts/security/pod-security-standards).
+Você pode configurar esse controlador de admissão para definir padrões em todo 
+o cluster e [exceções](/docs/concepts/security/pod-security-admission/#exemptions).
+
+## {{% heading "prerequisites" %}}
+
+{{% version-check %}}
+
+- Garanta que a `PodSecurity` do [`feature gate`](/docs/reference/command-line-tools-reference/feature-gates/#feature-gates-for-alpha-or-beta-features)
```
Comment (stormqueen1990) [inline content/pt-br/docs/tasks/configure-pod-container/enforce-standards-admission-controller.md:16]: "`PodSecurity` do feature gate" is reading a bit weird to me. What we want the reader to make sure is that the feature gate named `PodSecurity` is enabled. Maybe "o _feature gate_ `PodSecurity` está ativado"?

It's also better to format _feature gate_ as italic rather than code formatting.

**Review by stormqueen1990 — CHANGES_REQUESTED** [review]
Just one minor change, other than that LGTM!

**Comment by MrErlison** [comment]
Hi @DonatoHorn 

Your contribution is important to us Portuguese speakers and is well on its way. 

We are here to help you and open to answer any questions, criticism or suggestions. To do so, just access our channel #kubernetes-doc-pt on slack that is always available for you. 

We are waiting for your return.

---

Sua contribuição é importante para nós falantes da língua portuguesa e está bem encaminhada. 

Estamos aqui para ajudá-lo e abertos para sanar qualquer dúvida, crítica ou sugestão. Para isso basta acessar o nosso canal no slack #kubernetes-doc-pt que está sempre disponível para recebê-lo.

Estamos aguardando o seu retorno.

**Comment by stormqueen1990** [comment]
/retitle [pt-br] Add docs/tasks/configure-pod-container/enforce-standards-admission-controller.md
/lgtm

**Review by devlware — COMMENTED** [review]
/approve
